rasa/cli/shell.py

```diff
@@ -1,10 +1,22 @@
 import argparse
+import logging
+import os
+
 from typing import List
 
+import rasa.nlu.run
```

Comment:
let's import this in the `shell` function, we need to be a bit careful with imports in the `cli` package as any import will slow down the command line interface (esp, if one of these pulls in e.g. `tensorflow`)